### DISCUSSION TITLE
Youtube Revanced on Waydroid in Weston

### DISCUSSION THREAD

**Discussion by hardbrocklife** [waydroid/waydroid#913] [Q&A]
I have been tinkering around with Waydroid inside Weston on my Manjaro/KDE/X11 desktop.

I have waydroid working inside weston. I was able to install F-Droid, gmail, and Nextcloud because I couldnt figure out what folders where shared between the Weston/Waydroid instance the systems directories lol.

The reason for doing all of this though is to basically create a desktop yoututbe app using Revanced. I have microG install, downloaded the correct Youtube APK and patched it, but Revanced only pops up for a couple seconds before crashing. 

Just curious if anyone had an idea as to what my next step in troubleshooting would be. I dont see anything in the waydroid.log that indicates anything weird. Is it possibly and Arm vs X86_AMD64 thing?

**Comment by hoshi411** [comment]
Can't wait to get revanced working on waydroid.  This is the whole reason I installed waydroid

**Comment by nille02** [comment]
I have the same issue. without patching the app it works. even with just one patch enabled, it crashes.

**Comment by Flashwalker** [comment]
Installation fails
